src/js/arrays_loops.js

```diff
@@ -0,0 +1,131 @@
+import { contacts } from "../data/contacts.js";
+
+console.log(contacts);
+
+// Array Order Methods
+
+// 1.	Write a function that takes the array of contacts and returns a new array sorted by last name.
+
+const contactsArray = contacts.results;
+
+function sortedByLastName(contactsArray) {
+  return contactsArray.sort((a, b) => {
+    const lastnameA = a.name.last.toLowerCase();
+    const lastnameB = b.name.last.toLowerCase();
+
+    if (lastnameA > lastnameB) {
+      return 1;
+    }
+    if (lastnameA < lastnameB) {
+      return -1;
+    }
+    return 0;
+  });
+}
+
+console.log(sortedByLastName(contactsArray));
+
+// 2.	Write a function to return the array of contacts in reverse order. Do not use the built-in reverse() method.
+function reverseOrderArray(contactsArray) {
+  let contactObject = {};
```

Comment:
You are using a declaration here outside the needed scope
the declaration is also not needed.